packages/services/src/models/notification-scanners/NotificationScannerBaseModel.js

```diff
@@ -0,0 +1,34 @@
+const { env: { CLIENT_URL } } = process
+
+export default class NotificationScannerBaseModel {
+  async shouldNotifyUser(user) {
+    user = await user.$fetchGraph('[email, notificationSetting]')
+    return (
+      user?.email &&
+      !user?.notificationSetting?.notificationsDisabled &&
+      (user?.emailVerified || this._sendUnverified)
+    )
+  }
+  get _MINUTES() { return 60 * 1000 }
+  get _HOURS() { return 60 * this._MINUTES }
+  get _DAYS() { return 24 * this._HOURS }
+  get _CLIENT_URL() { return CLIENT_URL }
```

Comment:
Coming off of the other discussion of defining these as static methods, is there a reason to prefer internal getters over a constant import?

Reply:
in every derived class I just do `this` to reach them instead of importing a module every time. Since every derived class needs to define scan periods it felt like this should be part of the base